src/components/ChannelList/hooks/usePaginatedChannels.js

```diff
@@ -58,12 +56,12 @@ export const usePaginatedChannels = ({
       setHasNextPage(channelQueryResponse.length >= newOptions.limit);
       setOffset(newChannels.length);
     } catch (e) {
-      setLoadingChannels(false);
-      setLoadingNextPage(false);
-      setRefreshing(false);
```

Comment:
We have retry logic, so lets not set these variables to false just yet. We want to show loading indicator until we are trying queryChannels api.

Reply:
makes sense, good catch!